test/extensions/bootstrap/wasm/test_data/logging_rust.rs

```diff
@@ -0,0 +1,49 @@
+use log::{debug, error, info, trace, warn};
```

Comment:
This file is copied (not moved) from `test/extensions/bootstrap/wasm/test_data/logging_rust/src/lib.rs`, and it's currently unused.